contrib/release-notes/release-notes.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/github"
+)
+
+var (
+	target = flag.Int("last-release-pr", 0, "The PR number of the last versioned release.")
+)
+
+func main() {
+	flag.Parse()
+
+	if *target == 0 {
+		fmt.Printf("--last-release-pr is required.\n")
+		os.Exit(1)
+	}
+
+	client := github.NewClient(nil)
+
+	done := false
+
+	opts := github.PullRequestListOptions{
+		State:     "closed",
+		Sort:      "updated",
+		Direction: "desc",
+		ListOptions: github.ListOptions{
+			Page:    0,
+			PerPage: 100,
+		},
+	}
+
+	for !done {
+		opts.Page++
+		results, _, err := client.PullRequests.List("GoogleCloudPlatform", "kubernetes", &opts)
+		if err != nil {
+			fmt.Printf("Error contacting github: %v", err)
```

Comment:
should we exit here? this could easily get lost in the noise of a bunch of PRs and make it easy to miss changes